src/test/ansible-role-2.9/copy.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "name": "Copy file",
+        "copy": {
+            "src": "{{ item }}",
+            "dest": "/some/path",
+            "owner": "root",
+            "group": "root",
+            "mode": 509
```

Comment:
Note: JSON, unlike YAML, does not allow me to write this as an octal `0755`, so 509 is the decimal poor-mans-representation of this value here. :sweat_smile: 

(I guess there isn't any easy way to provide this test as a YAML file instead? It would make more sense in this case, since Ansible roles are typically authored as YAML in the first place.)

Reply:
Unfortunately not at this point. All tests are JSON

Reply:
:+1: I'll live with it.